pkgs/tools/networking/jnettop/0001-Use-64-bit-integers-for-byte-totals-support-bigger-u.patch

```diff
@@ -0,0 +1,104 @@
+From: Ari Pollak <aripollak@gmail.com>
```

Comment:
Sure thing, I found it on `sources.debian.net` but I'm not sure how stable that URL is.

Updated the commit.